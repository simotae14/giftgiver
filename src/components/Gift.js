import React, { Component } from 'react';
import { Form, Button } from 'react-bootstrap';

class Gift extends Component {
    constructor() {
        super();
        this.state = {
            person: '',
            present: ''
        };
    }
    render() {
        return(
            <div>
                <Form>
                    <Form.Group>
                       <Form.Label>Person</Form.Label>
                        <Form.Control
                            className='input-person'
                            onChange={event => this.setState({ person: event.target.value })}
                        />
                    </Form.Group>
                </Form>
            </div>
        );
    }
}

export default Gift;