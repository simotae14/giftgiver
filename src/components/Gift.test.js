import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Gift from './Gift';

configure({adapter: new Adapter()});

describe('Gift', () => {
    // create the shallow rendering
    const gift = shallow(<Gift />);

    it('renders properly', () => {
        expect(gift).toMatchSnapshot();
    });
});