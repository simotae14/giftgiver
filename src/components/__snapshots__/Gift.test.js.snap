// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gift renders properly 1`] = `
<div>
  <Form
    as="form"
    inline={false}
  >
    <FormGroup
      as="div"
    >
      <FormLabel
        column={false}
        srOnly={false}
      >
        Person
      </FormLabel>
      <FormControl
        as="input"
        className="input-person"
        onChange={[Function]}
      />
    </FormGroup>
  </Form>
</div>
`;
